stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -124,6 +131,8 @@ public class MessageListViewModel(
     private val dateSeparatorThresholdMillis: Long = TimeUnit.HOURS.toMillis(DateSeparatorDefaultHourThreshold),
     private val deletedMessageVisibility: DeletedMessageVisibility = DeletedMessageVisibility.ALWAYS_VISIBLE,
     private val messageFooterVisibility: MessageFooterVisibility = MessageFooterVisibility.WithTimeDifference(),
+    private val messageId: String? = null,
+    private val navigateToThreadViaNotification: Boolean = false,
```

Comment:
Maybe `navigateToMessageThread`? Could be used for opening a message inside a thread without a PN if users have message search capability?

Reply:
We currently don't have a good way of loading messages around a thread message in `v5` so this might not be the best approach as it could create incorrect behaviors.

However this is a good suggestion for a `v6` improvement.